app/src/main/java/org/zephyrsoft/trackworktime/timer/TimeCalculatorV2.java

```diff
@@ -553,4 +566,12 @@ public void calculatePeriod(PeriodEnum period, boolean includeFlexiTime) {
 		this.currentDayActual = currentDayActual;
 		this.currentDayTarget = currentDayTarget;
 	}
+
+	public static long roundUpToMultiple(long value, long multiple) {
+		if (multiple == 1) {
+			return value;
+		} else {
+			return (value + multiple - 1) / multiple * multiple;
```

Comment:
This won't work in all cases. Here's an example calculation for the inputs **value = 43, multiple = 15** which should round to 45.

But it doesn't, the method would return 57 with these inputs.

To be on the safe side, I'd recommend creating a unit test for the calulation method where some "normal" cases and many (if not all) of the edge cases are checked. For a start you can copy `TimeCalculatorTest` to `TimeCalculatorV2Test` and adjust the content accordingly.

Reply:
Possible values of `multiple` are: 1, 2, 3, 4, 5, 6, 10, 12, 15, 20, 30, 60

But probably only the values 5, 10, 15, 30 and 60 will be used in real life...

Reply:
Oh, that's weird. I'll write some tests for it.

Also, I do use 6 as a multiple myself since my work tracks time to an accuracy of 0.1 hours. Therefore I think it makes sense to leave it up to the user to choose any divisor of 60 :)